### PR TITLE
feat(chart): productionize helm chart

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

File: deployments/server/templates/configmap.yaml [hunk 1]
```diff
@@ -28,7 +27,7 @@ data:
       {{- toYaml .Values.global.usageSender | nindent 6 }}
     kubernetesManager:
       enableLeaderElection: {{ .Values.kubernetesManager.enableLeaderElection }}
-      leaderElectionID: {{ include "job-manager-dispatcher.fullname" . }}
+      leaderElectionID: {{ include "inference-manager-server.fullname" . }}
```
Comment (kkaneda) [inline deployments/server/templates/configmap.yaml:30]: thanks!

File: deployments/server/templates/deployment.yaml [hunk 2]
```diff
@@ -55,8 +55,18 @@ spec:
           containerPort: {{ .Values.adminPort }}
           protocol: TCP
         - name: km-metrics
-          containerPort: {{ .Values.kubernetesManagerMetricsPort }}
+          containerPort: {{ include "inference-manager-server.metricsPort" . }}
```
Comment (kkaneda) [inline deployments/server/templates/deployment.yaml:58]: I might be missing, but this is not defined?

**Review by kkaneda — APPROVED** [review]
LG!

File: deployments/server/templates/deployment.yaml (hunk 2 above)
Comment (Ladicle) [reply deployments/server/templates/deployment.yaml:58]: It's defined in the helper template.

Comment (kkaneda) [reply deployments/server/templates/deployment.yaml:58]: ah i see. nice!

**Pull request merged**